x/data/client/testsuite/suite.go

```diff
@@ -22,12 +22,6 @@ type IntegrationTestSuite struct {
 	signer             sdk.AccAddress
 }
 
-/*
-func NewIntegrationTestSuite(cfg network.Config) *IntegrationTestSuite {
-	return &IntegrationTestSuite{cfg : cfg}
-}
-*/
-
 // TODO call NewIntegrationTestSuite to set cfg field of IntegrationTestSuite
```

Comment:
should I remove all comments in suite.go or NewIntegrationTestSuite only ?

Reply:
Let's introduce minimal changes in this PR.